pkg/controller/servicemesh/memberroll/controller.go

```diff
@@ -31,7 +31,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-const controllerName = "controller_servicemeshmemberroll"
+const controllerName = "servicemeshmemberroll-controller"
```

Comment:
Should we use acronyms (e.g. "smmr-controller") here? If we use the full name, the log lines are very long even before you get to the actual log message.

Reply:
Oh, but the same name is also used in events. I'm not a big fan of using acronyms there. But maybe that's not a bad idea. Users working with the Maistra API will be familiar with the acronyms, since they'll probably use them in oc/kubectl commands.

Reply:
Let's stick with the long names for now; otherwise we create inconsistencies with e.g. `validatingwebhook-controller`